Geek-Collectors-Network/geek-collectors-network
# 30 add login page client side input validation

## Summary

Added client-side validation using `Yup` and `Formik`, identical to what's used on `RegistrationForm`.

## Testing

1. Navigate to the landing page using Deploy Preview (below)
2. Navigate to the `Login` page
3. Click `Submit` and confirm that the `email` and `password` fields outline in red and display error messages
4. Fill in the `email`, don't fill in `password`, click `Submit`, and confirm the password field goes red and displays an error
5. Fill in the `password`, don't fill in `email`, click `Submit`, and confirm the email field goes red and displays an error
6. Fill in both fields with valid inputs and click the `Submit`; confirm, in Google Developer Tools, that an object is created containing both the entered email and password.
